components/aws/sagemaker/tests/integration_tests/component_tests/test_v2_monitoring_schedule_component.py

```diff
@@ -0,0 +1,258 @@
+            os.path.join(test_file_dir, "config.yaml"),
+            os.path.join(download_dir, "config.yaml"),
+        )
+    )
+    k8s_client = ack_utils.k8s_client()
+
+    # parameters for model bias job definition
+    job_definition_name = (
+        utils.generate_random_string(10) + "-v2-model-bias-job-definition"
+    )
+    test_params["Arguments"]["job_definition_name"] = job_definition_name
+    test_params["Arguments"]["model_bias_job_input"]["endpointInput"][
+        "endpointName"
+    ] = deploy_endpoint
+
+    # parameters for monitoring schedule
+    monitoring_schedule_name = (
+        utils.generate_random_string(10) + "-v2-monitoring-schedule"
+    )
+    test_params["Arguments"]["monitoring_schedule_name"] = monitoring_schedule_name
+    test_params["Arguments"]["monitoring_schedule_config"][
+        "monitoringJobDefinitionName"
+    ] = job_definition_name
+
+    try:
+        _, _, _ = kfp_client_utils.compile_run_monitor_pipeline(
+            kfp_client,
+            experiment_id,
+            test_params["PipelineDefinition"],
+            test_params["Arguments"],
+            download_dir,
+            test_params["TestName"],
+            test_params["Timeout"],
+        )
+
+        job_definition_describe = ack_utils._get_resource(
+            k8s_client, job_definition_name, "modelbiasjobdefinitions"
+        )
+
+        # Check if the job definition is created
+        assert job_definition_describe["status"]["ackResourceMetadata"]["arn"] != None
+
+        # Verify if monitoring schedule is created with correct name and endpoint
+        monitoring_schedule_describe = sagemaker_client.describe_monitoring_schedule(
+            MonitoringScheduleName=monitoring_schedule_name
+        )
+        assert (
+            monitoring_schedule_name
+            in monitoring_schedule_describe["MonitoringScheduleArn"]
+        )
+
+        assert monitoring_schedule_describe["MonitoringScheduleStatus"] == "Scheduled"
+
+        assert monitoring_schedule_describe["EndpointName"] == deploy_endpoint
+
+    finally:
+        ack_utils._delete_resource(
+            k8s_client,
+            job_definition_name,
+            "modelbiasjobdefinitions",
+            wait_periods=10,
+            period_length=30,
+        )
+        ack_utils._delete_resource(
+            k8s_client,
+            monitoring_schedule_name,
+            "monitoringschedules",
+            wait_periods=10,
+            period_length=30,
+        )
+
+
+# Test updating monitoring schedule using the same pipeline
+# Steps:
+# Prepare pipeline inputs for job_definition_1 and monitoring schedule
+# Run the pipeline
+# Update pipeline input (instanceType) for job_definition_2
+# Rerun the same pipeline
+@pytest.mark.parametrize(
+    "test_file_dir",
+    [
+        pytest.param(
+            "resources/config/v2-monitoring-schedule-update",
+            marks=[pytest.mark.v2],
+        ),
+    ],
+)
+def test_v2_monitoring_schedule_update(
```

Comment:
this test covers everything that test_v2_monitoring_schedule does, shall we just keep this one?

Reply:
We could, but right now we seem to only include creating resources in canary.